src/app/components/planner-list/planner-list.component.ts

```diff
@@ -251,7 +256,20 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
       //this.treeList.update();
       this.prevWorkItemLength = this.workItems.length;
     }
+    if(this.toolbarHeight) {
+      let toolbarHt:any =  this.toolbarHeight.nativeElement.offsetHeight;
+      let quickaddHt:any =  this.quickaddHeight.nativeElement.offsetHeight;
+      let hdrHeight;
```

Comment:
`hdrHeight = 0` would make more sense. Using this variable for arithmetic operation.